### PR TITLE
converted text blocks to javascript strings

### PR DESCRIPTION
**Pull request by jbpayne**
This is a refactor from https://github.com/codeforbtv/cvoeo-app/pull/54 to change all text blocks to javascript strings.  There's also some blocks of code that changed a bit from automatic formatting.

### CODE REVIEW

File: screens/dashboard/index.js [hunk 1]
```diff
@@ -231,7 +231,7 @@ class Dashboard extends Component<Props> {
                 </View>
                 <View style={styles.smallerBlock}>
                   <Text style={styles.bigBlock}></Text>
-                  <Text style={[styles.money, styles.start]}>$500</Text>
+                  <Text style={[styles.money, styles.start]}>{'\$500'}</Text>
```
Comment (johnneed) [inline screens/dashboard/index.js]: I don't think you need to escape $.

File: screens/dashboard/index.js [hunk 2]
```diff
@@ -240,7 +240,7 @@ class Dashboard extends Component<Props> {
 
           <View style={styles.padding}>
             <View style={styles.goalsBox}>
-              <Text style={[styles.blockTitle, styles.goalsTitle]}>CURRENT GOALS:</Text>
+              <Text style={[styles.blockTitle, styles.goalsTitle]}>{'CURRENT GOALS\:'}</Text>
```
Comment (johnneed) [inline screens/dashboard/index.js]: I don't think you need to escape : either

File: screens/login/login-screen.js [hunk 3]
```diff
@@ -75,8 +75,8 @@ class LoginScreen extends Component<Props> {
                                 <TouchableHighlight
                                     style={styles.link}
                                     onPress={() => this.props.navigation.navigate('ForgotPassword')}
-                                    underlayColor="transparent">
-                                    <Text style={styles.linkText}>forgot password?</Text>
+                                    underlayColor='transparent'>
+                                    <Text style={styles.linkText}>{'forgot password\?'}</Text>
```
Comment (johnneed) [inline screens/login/login-screen.js]: question marks don't need to be escaped.

File: screens/dashboard/index.js [hunk 4]
```diff
@@ -169,14 +169,14 @@ class Dashboard extends Component<Props> {
           <View style={styles.padding}>
             <View style={styles.progressBox}>
               <View style={styles.spaceRow}>
-                <Text style={[styles.bigTitle, styles.bigLetters]}>${incentivesEarned}</Text>
+                <Text style={[styles.bigTitle, styles.bigLetters]}>{'$' + incentivesEarned}</Text>
                 <Text style={styles.bigBlock}></Text>
-                <Text style={styles.bigTitle}>{percentComplete}% Complete!</Text>
+                <Text style={styles.bigTitle}>{percentComplete+ '\% Complete\!'}</Text>
```
Comment (johnneed) [inline screens/dashboard/index.js]: % doesn't need to be escaped.

Comment (johnneed) [reply screens/dashboard/index.js]: neither does !

File: screens/dashboard/index.js [hunk 5]
```diff
@@ -169,14 +169,14 @@ class Dashboard extends Component<Props> {
...
               </View>
               <View style={styles.dashRow}>
                 <View style={styles.smallerBlock}>
                   <Text style={styles.bigBlock}></Text>
-                  <Text style={[styles.money, styles.end]}>$0</Text>
+                  <Text style={[styles.money, styles.end]}>{'\$0'}</Text>
```
Comment (johnneed) [inline screens/dashboard/index.js]: $ doesn't need to be escaped.

**Comment by jbpayne** [comment]
Thank you for catching this.  That was a silly mistake. (10 silly mistakes?)

**Pull request merged**